variables.tf

```diff
@@ -10,48 +10,57 @@ variable "scope" {
 
 variable "managed_rules" {
   type = list(object({
-    name            = string
-    priority        = number
-    override_action = string
-    excluded_rules  = list(string)
+    name                 = string
+    priority             = number
+    override_action      = string
+    excluded_rules       = list(string)
+    scope_down_statement = statement
```

Comment:
This type definition is definitely wrong. But I'm not sure how to correct it? It seems like [Statement](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_rule_group#statement) is recursively defined.

Reply:
Statement isn't a valid type; considering `excluded_rules` is a `list(string)` type and the structure should be similar, I suspect what you want is a `list(string)`